src/tiled/mapdocumentactionhandler.cpp

```diff
@@ -175,6 +180,7 @@ MapDocumentActionHandler::~MapDocumentActionHandler()
 
 void MapDocumentActionHandler::retranslateUi()
 {
+    mActionGotoTile->setText(tr("&Go to"));
```

Comment:
At least as long as this action can only go to tiles, I would change the text to `&Go to Tile...` (note the added `...` because it spawns a dialog).